plugins/woocommerce/client/admin/client/core-profiler/pages/Plugins/components/plugin-error-banner/PluginErrorBanner.tsx

```diff
@@ -25,7 +25,7 @@ export const PluginErrorBanner = ( {
 		case pluginsInstallationPermissionsFailure:
 		case pluginsInstallationErrors?.some(
 			// it really shouldn't get here since permissions are pre-checked. but we'll check for 403 just to be safe.
-			( e ) => e.errorDetails?.data.data.status === 403 // 403 is the code representing rest_authorization_required_code()
+			( e ) => e.errorDetails?.data?.data?.status === 403 // 403 is the code representing rest_authorization_required_code()
```

Comment:
When it's not 403, such as cases when plugin slug is invalid, this fatals

Reply:
oooooooh